### ISSUE TITLE
Fixed-X knockoffs X.ko'X.ko

### ISSUE THREAD

**Issue by nignatiadis** [biona001/Knockoffs.jl#70]
Hi Benjamin,

I hope you are doing well! 

I wanted to check about the following. The following seems not to match (for any of the fixed-X knockoff constructions):

```julia
me = fixed_knockoffs(X, :maxent)
maximum(abs.(me.Xko'me.Xko .- me.Sigma)) #large 
```
Do you know why that is? (Happy to provide a MWE if it helps, though it happens for all designs I have tried it on.)

Thank you,
Nikos

**Comment by biona001** [comment]
@nignatiadis sorry for the delayed response, somehow this issue fell under the radar. 

It'll be fixed on master soon.

**Comment by biona001** [comment]
@nignatiadis I think the issue should be fixed on v1.1.9. It'll be released once CI passes
```julia
using Knockoffs # v1.1.9
using Random
using LinearAlgebra
X = randn(1000, 200)

me = fixed_knockoffs(X, :maxent)
maximum(abs.(me.Xko'me.Xko .- me.Sigma)) #5.065392549852277e-14 
```

Please let me know if you run into any other problems, and sorry again for the delayed response.
